examples/traverse_graph.rs

```diff
@@ -0,0 +1,135 @@
+use dot_vox::{DotVoxData, Model, Rotation, SceneNode};
+use glam::Vec3;
+
+fn iterate_vox_tree<F: FnMut(&Model, &Vec3, &Rotation) -> ()>(vox_tree: &DotVoxData, mut fun: F) {
```

Comment:
nit: you can just write 
```suggestion
fn iterate_vox_tree(vox_tree: &DotVoxData, mut f: impl FnMut(&Model, &Vec3, &Rotation)) {
```